File: tests/bench/table_heavy.lua
local t = {}
local i = 0
while i < 10000 do
    t[i] = i * i
    i = i + 1
end
i = 0
local sum = 0
while i < 10000 do
    sum = sum + t[i]
    i = i + 1
end
print(sum)

File: tests/bench/node/table_heavy.js
let t = {};
for (let i = 0; i < 100000; i++) {
    t["key" + i] = i * i;
}
let sum = 0;
for (let i = 0; i < 100000; i++) {
    sum += t["key" + i];
}
console.log(sum);
